_includes/v19.1/app/insecure/basic-sample.clj

```diff
@@ -0,0 +1,31 @@
+(ns test.test
+  (:require [clojure.java.jdbc :as j]
+            [test.util :as util]))
+
+;; Define the connection parameters to the cluster.
+(def db-spec {:subprotocol "postgresql"
+              :subname "//localhost:26257/bank"
+              :user "maxroach"
+              :password ""})
+
+(defn test-basic []
+  ;; Connect to the cluster and run the code below with
+  ;; the connection object bound to 'conn'.
+  (j/with-db-connection [conn db-spec]
+
+    ;; Insert two rows into the "accounts" table.
+    (j/insert! conn :accounts {:id 1 :balance 1000})
+    (j/insert! conn :accounts {:id 2 :balance 250})
+
+    ;; Print out the balances.
+    (println "Initial balances:")
+    (->> (j/query conn ["SELECT id, balance FROM accounts"])
+         (map println)
+         doall)
+
+    ;; The database connection is automatically closed by with-db-connection.
```

Comment:
It's not clear this comment is really needed, since in Lisp world `WITH-FOO` always implies that `FOO` is a resource of some kind that will be auto-closed if the body to be executed fails.  The comment also pushes the closing parens onto their own line, which is also not generally good Lisp style.  (But for the record these issues are not really hurting anything, so feel free to leave this stuff in.  I'm just being a Lisp nerd because I get so few chances to do so here.)

Reply:
Let's take it out then!